src/modules/data/blocks/venue/subscribers.js

```diff
@@ -14,6 +14,7 @@ import {
 	actions as venueActions,
 	selectors as venueSelectors,
 } from '@moderntribe/events/data/blocks/venue';
+import { syncVenuesWithPost } from '@moderntribe/events/blocks/event-venue/data/meta-sync';
```

Comment:
This is the new implementation that I was talking about [here](https://github.com/the-events-calendar/the-events-calendar/pull/4345/files#r1283818930)